language/tools/move-mv-llvm-compiler/src/main.rs

```diff
@@ -165,21 +169,43 @@ fn main() -> anyhow::Result<()> {
             tgt_platform.llvm_cpu(),
             tgt_platform.llvm_features(),
         );
-        let mod_id = global_env
-            .get_modules()
-            .last()
-            .map(|m| m.get_id())
-            .expect(".");
         let global_cx = GlobalContext::new(&global_env, tgt_platform, &llmachine);
-        let modname = global_env.get_module(mod_id).llvm_module_name();
-        let mut llmod = global_cx.llvm_cx.create_module(&modname);
-        let mod_cx = global_cx.create_module_context(mod_id, &llmod, &args);
-        mod_cx.translate();
-        if !args.obj {
-            llvm_write_to_file(llmod.as_mut(), args.llvm_ir, &args.output_file_path)?;
-            drop(llmod);
-        } else {
-            write_object_file(llmod, &llmachine, &args.output_file_path)?;
+
+        let args = Args::parse();
```

Comment:
Why did you re-introduce this redundant call to `Args::parse()`? Redundant parses had been removed in a recent patch.

Reply:
Overlooked, just copied it from my old patch... 
Will fix.